cli/testdata/coder_server_--help.golden

```diff
@@ -80,6 +80,18 @@ Use a YAML configuration file when your server launch become unwieldy.
           
           Write out the current server config as YAML to stdout.
 
+INTROSPECTION / HEALTHCHECK OPTIONS: 
+      --healthcheck-refresh duration, $CODER_HEALTHCHECK_REFRESH (default: 10m0s)
+          Refresh interval for healthchecks.
+
+      --threshold-database duration, $CODER_HEALTHCHECK_THRESHOLD_DATABASE (default: 15ms)
```

Comment:
```suggestion
      --healthcheck-database-threshold duration, $CODER_HEALTHCHECK_THRESHOLD_DATABASE (default: 15ms)
```

Suggestion: unified prefix and reorder so future flags can be grouped together (--hc-db-query-latency, etc). Maybe we never add anything but it's nice to be prepared.

Reply:
This was an oversight. Fixed 👍